packages/core/admin/admin/src/components/ConfigurationsProvider/reducer.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable consistent-return */
+/*
+ *
+ * ConfigurationsProvider reducer
+ *
+ */
+
+import produce from 'immer';
+
+const initialState = {
+  customMenuLogo: null,
```

Comment:
WDYT about the following state:

```
logos: {
  auth: { custom, default },
  menu: { custom, default }
}, 
```

Reply:
mmh idk
- is it recommended/useful to add a key in a reducer state that won't change dynamically (e.g. default won't change dynamically in-app)
- about wrapping logo values in logos key, I was wondering if I should "prepare" the field for auth logo since it's not scheduled yet?
WDYT @soupette ?

Reply:
Yes you are right in fact you just need: 

```
logos: { menu, auth }
```